Jellyfin.Server/Program.cs

```diff
@@ -175,6 +176,11 @@ private static async Task StartServer(IServerApplicationPaths appPaths, StartupO
 
                 // Re-use the host service provider in the app host since ASP.NET doesn't allow a custom service collection.
                 appHost.ServiceProvider = _jellyfinHost.Services;
+
+                var factory = appHost.ServiceProvider.GetRequiredService<IDbContextFactory<JellyfinDbContext>>();
+                var provider = appHost.ServiceProvider.GetRequiredService<IJellyfinDatabaseProvider>();
+                provider.DbContextFactory = factory;
```

Comment:
Since these 3 lines are in 3 places (also ApplicationHost.InitializeServices) we should create a helper to reduce the chance we miss a place if we need to update them

Reply:
done